src/Microsoft.AspNetCore.Blazor/Json/JsonUtil.cs

```diff
@@ -13,13 +13,22 @@ namespace Microsoft.AspNetCore.Blazor
     public static class JsonUtil
     {
         /// <summary>
-        /// Serializes the value as a JSON string.
+        /// Serializes the value as a JSON string using the default property naming scheme <see cref="PropertyNaming.CamelCase"/>.
         /// </summary>
         /// <param name="value">The value to serialize.</param>
         /// <returns>The JSON string.</returns>
         public static string Serialize(object value)
             => SimpleJson.SimpleJson.SerializeObject(value);
 
+        /// <summary>
+        /// Serializes the value as a JSON string with the specified property naming scheme.
+        /// </summary>
+        /// <param name="value">The value to serialize.</param>
+        /// <param name="naming">The property naming scheme for serializing.</param>
+        /// <returns>The JSON string.</returns>
+        public static string Serialize(object value, PropertyNaming naming)
+            => SimpleJson.SimpleJson.SerializeObject(value, (int)naming);
```

Comment:
As per other comment, hopefully we can change the param type to `PropertyNaming` so there's no `(int)` cast here.